lib/chef/provider.rb

```diff
@@ -390,9 +390,6 @@ def action_#{name}
           end
         end
       end
-
-      require "chef/dsl/recipe"
-      include Chef::DSL::Recipe
```

Comment:
Okay, but a `libraries/` based provider using `use_inline_resources` would see a behavior change I think. We should make sure that ends up in the changelog at a minimum.